chaincode-labs/chaincode-podcast/tracepoints-and-monitoring-the-bitcoin-network.md

```diff
@@ -405,269 +405,269 @@ And we can hook into this trace point and extract, for example, which peer we se
-That's the goal, yeah.
+That’s the goal, yeah.
 
-Speaker 1: 00:18:02
+Mark Erhardt: 00:18:02
 
-Oh, I could see enterprises being super interested in having a closer look at what their nodes are doing, how they're connected and that sort of thing.
+Oh, I could see enterprises being super interested in having a closer look at what their nodes are doing, how they’re connected and that sort of thing.
 And it enables people to donate their logs more easily, maybe.
 Probably, yeah.
 If we see fun stuff, like what we talked to Martin about.
 Yeah.
 
-Speaker 0: 00:18:19
+Adam Jonas: 00:18:19
 
 Do you want to explain what that means?
 
-Speaker 1: 00:18:21
+Mark Erhardt: 00:18:21
 
 Yeah.
-I think that episode's not out yet.
+I think that episode’s not out yet.
 Right.
 So we had Martin Zum Sande visit us and we talked to him a little bit about how changing the peer to peer behavior could change the emergent behavior of the network in the whole.
 And there were some interesting events in the peer to peer sphere a year ago or so, where somebody started broadcasting vast amounts of made up addresses.
 
-Speaker 2: 00:18:45
+0xB10C: 00:18:45
 
 Oh, yeah.
 
-Speaker 1: 00:18:45
+Mark Erhardt: 00:18:45
 
 And we dissected that story a little bit with Martin and looked a little bit at the peculiarities of the address relay, how sending small chunks would carry further than sending big buckets of them and things like that.
 
-Speaker 2: 00:19:00
+0xB10C: 00:19:00
 
 Yeah, right.
 Actually, the address flooding that happened last summer actually was one of the motivations for that project.
 So one goal is actually to take some of the address flooding or similar flooding or anything in that direction and have an alert because I think, at least from my perspective, I was not directly involved in that.
 But from my perspective, this was just a coincidence that we actually saw that happening on the network and might be valuable for us to know something like that.
 
-Speaker 0: 00:19:28
+Adam Jonas: 00:19:28
 
-Yeah, it was, I think It was raised to GMAX's attention on a BitcoinTalk forum, which then got relayed to more active folks.
-But without someone combing BitcoinTalk forum, it wouldn't necessarily...
+Yeah, it was, I think It was raised to GMAX’s attention on a BitcoinTalk forum, which then got relayed to more active folks.
+But without someone combing BitcoinTalk forum, it wouldn’t necessarily…
 
-Speaker 2: 00:19:43
+0xB10C: 00:19:43
 
 In the end, there was a paper about it, and so it definitely got attention.
 But I think it could have happened that nobody reported it or the core development process, people involved in that never heard about it.
 
-Speaker 1: 00:19:55
+Mark Erhardt: 00:19:55
 
 Yeah.
 Making peer-to-peer traffic more readable and more accessible to regular users would maybe enable more of that to come to attention.
 
-Speaker 2: 00:20:04
+0xB10C: 00:20:04
 
 Yeah, and there are different attacks on the PTP networks.
```

Comment:
This is the type of AI transcription mistakes that we expect from our reviewers to catch.
```suggestion
Yeah, and there are different attacks on the P2P networks.
```